### DISCUSSION TITLE
Lazy loading multiple components

### DISCUSSION THREAD

**Discussion by ercsctt** [livewire/livewire#6147] [3. Feature ideas]
Hey,

Been playing around with V3 attempting to build a simple SPA and ran across a small "issue". It's actually not necessarily an issue, but I'd argue it's unexpected behavior, and as far as I see, no way around it right now.

When loading multiple components lazily on a page, the request bundler seems to also bundle together all lazy requests. Some components may take longer than others, so the request has to wait for the slowest component to finish before it can return the full response. 

My suggestion here is that any component with the `lazy` attribute should not have to be bundled, or another way of doing this could be introducing a `lazy.force` or `single` attribute which would bypass bundling and make multiple requests if necessary.

Feedback and suggestions are much appreciated, I can look into implementing this on my own but obviously fairly limited knowledge of the inner workings.

**Comment by joshhanley** [comment]
@ercsctt thanks for the suggestion! Any chance you could submit a failing test PR that covers this case so we can look into it? Thanks!

**Comment by lucasclgibson** [comment]
I feel like this should be a much higher priority but that may just be me, bundling all subsequent requests together defeats the object of lazy loading entirely.

Wonder what the best solution would be for this? Whatever it is, I think that it should be the default behaviour that each "lazy" component is treated individually with no need for an additional attribute.

🤞🤞

**Comment by eilerth** [comment]
I also found this to be an odd change in behavior as I was upgrading from livewire v2.
I've submitted this PR as a potential solution to the problem: https://github.com/livewire/livewire/pull/7102

Essentially, it allows you to add "isolate" to a livewire component to prevent it from bundling it's requests with other livewire components.
`<livewire:my-component isolate />`

**Comment by KuenzelIT** [comment]
I am writing a dashboard with multiple components where each components needs a different amount of time with external APIs. It would be nice to disable the bundling or to isolate components as suggested.
